sync/_databricks.py

```diff
@@ -672,20 +762,32 @@ def get_project_cluster_settings(project_id: str, region_name: str = None) -> Re
             }
         }
 
-        s3_url = project.get("s3_url")
-        if s3_url:
+        cluster_log_dest = urlparse(project.get("cluster_log_dest"))
+        if cluster_log_dest.scheme == "s3":
             result.update(
                 {
                     "cluster_log_conf": {
                         "s3": {
-                            "destination": f"{s3_url}/{project_id}",
+                            "destination": f"{cluster_log_dest.geturl()}/{project_id}",
                             "enable_encryption": True,
                             "region": region_name or boto.client("s3").meta.region_name,
                             "canned_acl": "bucket-owner-full-control",
                         }
                     }
                 }
             )
+
+        if cluster_log_dest.scheme == "dbfs":
```

Comment:
`elif` is probably better here

Reply:
Shouldn't ever be able to satisfy both conditions, but changed for safety sake